### PR TITLE
Rc users components1

### PR DESCRIPTION
**Pull request by robert-czwartosz**
Hi, I have done Manage Users view.
In storybook are stories for fake users database and our MongoDB.

### CODE REVIEW

File: frontend/src/components/ManageUsers/ManageUsers.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  useEffect(() => {
+    dispatch(sortData({table: 'Users', column: sortBy }));
+  }, [sortBy]);
+  useEffect(() => {
+    const f = {
+      column: /^[0-9a-fA-F]{1,16}$/.test(search) ? 'id' : 'surname',
+      values: [ search ],
+    }
+    dispatch(filterData({table: 'Users', filters: [ f ]}));
+  }, [search]);
```
Comment (fszwedo) [inline frontend/src/components/ManageUsers/ManageUsers.tsx:89]: if you will put one string in search and then remove it then the filter function returns empty results set - this is something probably to be slightly changed in the filterData function

easy workaround - just applying any filter brings the users back

Comment (fszwedo) [reply frontend/src/components/ManageUsers/ManageUsers.tsx:89]: nevermind, I had some old version of the branch locally, after I went to the table slice I spotted it's already fixed :)

**Review by fszwedo — APPROVED** [review]
apart from the things I thought I saw but were already fixed I have no comments, everything looks and works really good

**Comment by skorpiom** [comment]
Hey,
looks nice 
just one thing - that add button was deleted from the 'manage users' design because we gave up that funcionality ;)

**Review by skorpiom — APPROVED** [review]

**Pull request merged**